app/controllers/api/v1/tags_controller.rb

```diff
@@ -6,6 +6,15 @@ def index
     else
       @tags = Tag.all
     end
-    render json: @tags
+      render json: @tags
+   end
```

Comment:
The spacing here is kinda messy. I think we want something formatted closer to this:
```ruby
class controller
  def action
    if condition
      # code
    else
      # more code
    end

    # render
  end

  def other_action
    #code
  end
end
```